Model/Transaction.php

```diff
@@ -238,8 +238,13 @@ protected function buildLineItems($order, $items, $type = 'order') {
             ];
 
             if ($type == 'refund' && method_exists($item, 'getOrderItem')) {
```

Comment:
Should this be checking for the existence of 'getOrderItemId' now?

Reply:
@ryanreeves-taxjar Checking this out